CONTRIBUTING.md

```diff
@@ -50,3 +50,21 @@ immediately. These are good ones to tackle to help us actively fix bugs.
 Maintenance branches are how we manage the different supported point releases
 of RSpec. As such, while they might look like good candidates to merge into
 master, please do not open pull requests to merge them.
+
+## How do the cukes work?
```

Comment:
Is cukes a commonly enough understood colloquialism?

Reply:
I think with the next sentence _"The cucumber features for RSpec rails document how it works"_ it's ok.